src/metabase/search/settings.clj

```diff
@@ -15,10 +17,19 @@
 
 (defsetting search-engine
   (i18n/deferred-tru "Which engine to use by default when performing search. Supported values are :in-place, :appdb, and :semantic")
-  :visibility :internal
-  :export?    false
+  :visibility :authenticated
+  :export?    true
```

Comment:
I didn't immediately find docs on this, so not sure if these are the right values to use.